spec/rubocop/cop/layout/multiline_method_call_indentation_spec.rb

```diff
@@ -881,7 +881,7 @@ def a
         expect_no_offenses(<<~RUBY)
           return #{keyword} receiver.nil? &&
               !args.empty? &&
-              BLACKLIST.include?(method_name)
+              DENYLIST.include?(method_name)
```

Comment:
I can't see where's the actual constant definition that has to be renamed.

Reply:
Ah, that's an example. My bad! Use something like "FORBIDDEN_METHODS" or whatever, please.